Description: Temporarily disable rkyv
 Nothing in Debian currently depends on simba+rkyv, so temporarily disable to
 allow upgrading rkyv to 0.8.
 .
 There are some complications on ensuring rkyv::Portable for internal data
 structures.
Author: Blair Noctis <ncts@debian.org>
Forwarded: not-needed
Last-Update: 2025-01-20
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -83 +83 @@
-[dependencies.rkyv]
+[disabled.dependencies.rkyv]
@@ -105 +105 @@
-rkyv-serialize = ["rkyv"]
+#rkyv-serialize = ["rkyv"]
